examples/external-data/src/mock-customer-service/service.ts

```diff
@@ -229,6 +279,60 @@ export async function initializeCustomerService(props: ServiceProps): Promise<Se
 		result.send();
 	});
 
+	/**
+	 * An 'events' endpoint that can be called by Fluid services.
+	 * Currently, the only supported request type is 'session-end' {@link SessionEndEventsListenerRequest} which enables a the fluid service to notify this service
```

Comment:
Nit: I would recommend moving these contents into a `@remarks` block, since they don't describe the semantics of the endpoint.
```suggestion
	 * @remarks
	 * 
	 * Currently, the only supported request type is 'session-end' {@link SessionEndEventsListenerRequest} which enables a the fluid service to notify this service
```